orgparse/__init__.py

```diff
@@ -130,16 +129,31 @@ def load(path: Union[str, Path, TextIO], env: Optional[OrgEnv]=None) -> OrgNode:
 
     """
     orgfile: TextIO
```

Comment:
I don't understand what this line is about.

Reply:
this is just a type annotation, but probably not really necessary anymore